// @expo-google-fonts/nunito-sans/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const NunitoSans_200ExtraLight = require('./200ExtraLight/NunitoSans_200ExtraLight.ttf');
export const NunitoSans_300Light = require('./300Light/NunitoSans_300Light.ttf');
export const NunitoSans_400Regular = require('./400Regular/NunitoSans_400Regular.ttf');
export const NunitoSans_500Medium = require('./500Medium/NunitoSans_500Medium.ttf');
export const NunitoSans_600SemiBold = require('./600SemiBold/NunitoSans_600SemiBold.ttf');
export const NunitoSans_700Bold = require('./700Bold/NunitoSans_700Bold.ttf');
export const NunitoSans_800ExtraBold = require('./800ExtraBold/NunitoSans_800ExtraBold.ttf');
export const NunitoSans_900Black = require('./900Black/NunitoSans_900Black.ttf');
export const NunitoSans_200ExtraLight_Italic = require('./200ExtraLight_Italic/NunitoSans_200ExtraLight_Italic.ttf');
export const NunitoSans_300Light_Italic = require('./300Light_Italic/NunitoSans_300Light_Italic.ttf');
export const NunitoSans_400Regular_Italic = require('./400Regular_Italic/NunitoSans_400Regular_Italic.ttf');
export const NunitoSans_500Medium_Italic = require('./500Medium_Italic/NunitoSans_500Medium_Italic.ttf');
export const NunitoSans_600SemiBold_Italic = require('./600SemiBold_Italic/NunitoSans_600SemiBold_Italic.ttf');
export const NunitoSans_700Bold_Italic = require('./700Bold_Italic/NunitoSans_700Bold_Italic.ttf');
export const NunitoSans_800ExtraBold_Italic = require('./800ExtraBold_Italic/NunitoSans_800ExtraBold_Italic.ttf');
export const NunitoSans_900Black_Italic = require('./900Black_Italic/NunitoSans_900Black_Italic.ttf');
